src/tomczyk-kamil/Fsd.Kamil.Cs/Fsd.Kamil.Cs.Ex2-3/Fsd.Kamil.Cs.Ex2-3/Animals/Fox.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Kamil.Cs.Ex2_3.Animals
+{
+    public class Fox : Animal
+    {
+        public string VoiceOfAnimal = "Ring Ding Ding Ding Ding";
+
+        public Fox(string name, int number)
+            : base(name, number)
+        {
+        }
+
+        List<Fox> foxList = new List<Fox>();
```

Comment:
nie o to chodzi w tym zadaniu. Należało zrobić kilka klas zwierząt i jedną klasę w której będzie lista wszystkich możliwych zwierzaków. Dzięki polimorfizmowi moglibysmy w 1 kolekcji (np. tablica) przechowywać dowolne typy zwierząt. Robiliśmy coś podobnego na zajęciach. Musisz poprawić to zadanie